google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -67,6 +68,12 @@
  *     {@code java.log}).
  * <li>{@code com.google.cloud.logging.LoggingHandler.level} specifies the default level for the
  *     handler (defaults to {@code Level.INFO}).
+ *     <p> This configuration also sets the "base severity level".
```

Comment:
I feel like the information is useful, but at the user level it is a distraction since it can't be controlled. Could we move it somewhere where a maintainer would find it?